### PR TITLE
EVG-7274: restartTask mutation

### PR DESCRIPTION
**Pull request by SupaJoon**
https://jira.mongodb.org/browse/EVG-7260
Will be used for the restart task button on the Task page

### CODE REVIEW

File: graphql/util.go [hunk 1]
```diff
@@ -259,3 +259,17 @@ func (p *PatchVariantsTasksRequest) BuildFromGqlInput(r PatchReconfigure) {
 		p.VariantsTasks = append(p.VariantsTasks, variantTasks)
 	}
 }
+
+// GetAPITaskFromTask builds an APITask from the given task
+func GetAPITaskFromTask(ctx context.Context, sc data.Connector, task task.Task) (*restModel.APITask, error) {
```
Comment (tgrander) [inline graphql/util.go:264]: +1

File: globals.go [hunk 2]
```diff
@@ -226,6 +226,7 @@ func IsFailedTaskStatus(status string) bool {
 // evergreen package names
 const (
 	UIPackage      = "EVERGREEN_UI"
+	GQLPackage     = "EVERGREEN_GQL"
```
Comment (john-m-liu) [inline globals.go:229]: There's no need to add this really, I would just use UIPackage. It's to differentiate between things that are user vs. system-initiated

File: public/static/js/subscriptions.js [hunk 3]
```diff
@@ -1,52 +1,52 @@
-const SUBSCRIPTION_JIRA_COMMENT = 'jira-comment';
-const SUBSCRIPTION_JIRA_ISSUE = 'jira-issue';
-const SUBSCRIPTION_SLACK = 'slack';
-const SUBSCRIPTION_EMAIL = 'email';
-const SUBSCRIPTION_EVERGREEN_WEBHOOK = 'evergreen-webhook';
+const SUBSCRIPTION_JIRA_COMMENT = "jira-comment";
```
Comment (john-m-liu) [inline public/static/js/subscriptions.js:1]: Did you mean to format this file? It's fine that you did, just wanted to double check

Comment (SupaJoon) [reply public/static/js/subscriptions.js:1]: oops, no i didnt!! thank you!

**Pull request merged**